lib/active_model/serializer/include_tree.rb

```diff
@@ -0,0 +1,77 @@
+module ActiveModel
+  class Serializer
+    # Description
+
+    class IncludeTree
+      class << self
+        # Builds an IncludeTree from a comma separated list of dot separated paths (JSON API format).
+        # @example `'posts.author, posts.comments.upvotes, posts.comments.author'`
+        #
+        # @param [String] included
+        # @return [IncludeTree]
+        #
```

Comment:
Not sure, there are many places in the current code base where the format is as above. I don't mind changing it, once we make a consistent choice for the future.